js/repl/loadBabel.js

```diff
@@ -77,5 +78,9 @@ export default async function loadBabel(
     version = DEFAULT_BABEL_VERSION;
   }
 
-  return doLoad(`https://unpkg.com/babel-standalone@${version}/babel.min.js`);
+  const babelStandalone = semver.gte(version, "7.0.0-beta.5")
```

Comment:
we can just add check for validity.
```js
semver.valid(version) && semver.gte(version, "7.0.0-beta.5")
  ? "@babel/standalone"
  : "babel-standalone";
```

Reply:
gonna check it later, I've assumed that if the `6` is accepted as a range it will be accepted as version too

thanks for checking